class Error(Exception):
    pass

class TriggerDone(Error):
    pass
